ui/design-system-core/src/components/MessageCard/index.tsx

```diff
@@ -30,7 +30,7 @@ const MessageCard: React.FC<MessageCardProps> = ({
           <Text variant={titleVariant} align="center" customStyle="grow">
             {title}
           </Text>
-          <Button onClick={onClose} plain>
+          <Button onClick={onClose} plain data-testid="close-icon">
```

Comment:
I suggest in such cases to use `aria-label` as `aria-label='close'` then target it on tests as `getByRole('button', {name: /close/i})`

Reply:
It's recommended that querying by test by id's should be the last resort where all the other options can't be applied like it's suggested [here](https://testing-library.com/docs/queries/about/#priority).